src/shared/sleep-config.h

```diff
@@ -67,6 +67,7 @@ int estimate_battery_discharge_rate_per_hour(
                 usec_t after_timestamp);
 int check_wakeup_type(void);
 int battery_trip_point_alarm_exists(void);
+int freeze_thaw_user_slice(const char **method);
```

Comment:
no need to export this, make the function static

Reply:
ok.